.github/workflows/daily-builds.yml

```diff
@@ -53,8 +53,19 @@ jobs:
           cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=Debug -GNinja -DCMAKE_CXX_COMPILER_LAUNCHER=ccache ${{ matrix.flags }}
           cd ${{github.workspace}}/build && pwd
           du -hcs _deps/
-      - name: Build
+      - name: Build & Run
         run: |
           cd ${{github.workspace}}/build
           ninja src/all
           ccache --show-stats
+          ./dragonfly --alsologtostderr --default_lua_config=allow-undeclared-keys
```

Comment:
can you show me the log of the example run for this action?

Reply:
It is changed in my recent changes - https://github.com/Abhra303/dragonfly/actions/runs/5447606193/workflow. There are still some issues regarding the installation of NodeJs. Once that fixed, I will push the newest change in this branch.